packages/components/MenuButton/src/MenuButton.tsx

```diff
@@ -106,6 +104,7 @@ export const MenuButton = compose<MenuButtonType>({
     primaryButton: { slotType: PrimaryButton as React.ComponentType },
     contextualMenu: { slotType: ContextualMenu as React.ComponentType },
     contextualMenuItems: React.Fragment,
+    svg: Svg
```

Comment:
Could we do something like what @lenahong did in ContextualMenu, specify this as "SVGXML" and just pass the full XML instead of needing function that returns JSX? That might get around the test app freezing.

I mention to see if we can keep the "endIcon" prop and also use an inline svg

Reply:
I tried both solutions before. With SVGXML colors didn't work. Though I can play with it to make it work. Current solution is implemented similar to Activity Indicator. It doesn't cause Tester freezing.

Reply:
Since this is a blocking issue I'm going to go with the current implementation and log an issue to come back to this.